tasks/main.yml

```diff
@@ -65,6 +65,45 @@
   delegate_to: "{{ hyper }}"
   # End of block
 
+- name: experimental Passthrough pci block
```

Comment:
I think this block deserves it's own `import_tasks:`. The name could be changed to something like `Experimental, find PCIe devices if necessary`, since as a user it might be scary to get a "experimental" "passhtrough" "pci" if I just want to launch a VM.